### PR TITLE
Index Creation Bugfixes

### PR DESCRIPTION
**Pull request by scbedd**
- Porting updates for azure-mgmt-web namespaces. 
- Updating index creation script to include replace relative links step.
- Other small updates to package_service_mapping.json 
- Updating documentation landing page with appropriate link text and location.

@Azure/azure-sdk-eng

### CODE REVIEW

File: eng/pipelines/create-doc-index.yml [hunk 1]
```diff
@@ -27,10 +27,17 @@ jobs:
         inputs:
           versionSpec: $(PythonVersion)
 
+      - template: eng/pipelines/templates/scripts/replace-relative-links.yml@azure-sdk-tools
+        parameters:
+          TargetFolder: $(Build.SourcesDirectory)
```
Comment (weshaggard) [inline eng/pipelines/create-doc-index.yml]: Out of curiosity some of the other repo's have passed `.` here in the context of these parameters which one is preferred?

**Comment by weshaggard** [comment]
Once this change goes in and you regenerate to doc index can you have a look over the broken links left in python github.io, you can use the pipeline at https://dev.azure.com/azure-sdk/internal/_build/results?buildId=225016&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=e73d691f-7b26-5ce3-d518-700413f3ba2c.

File: eng/pipelines/create-doc-index.yml (hunk 1 above)
Comment (scbedd) [reply eng/pipelines/create-doc-index.yml]: I'm actually validating the build results, and I'm almost positive I'll need to add the `.` explicitly. Will get back to you on this.

Comment (scbedd) [reply eng/pipelines/create-doc-index.yml]: Definitely necessary. Update pushed.

image

**Comment by scbedd** [comment]
> Once this change goes in and you regenerate to doc index can you have a look over the broken links left in python github.io, you can use the pipeline at https://dev.azure.com/azure-sdk/internal/_build/results?buildId=225016&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=e73d691f-7b26-5ce3-d518-700413f3ba2c.

I've eliminated both "true" 404s in the docs by updating the link location.
For the last remaining two issues, I'll push additional fixes to resolve them to this PR before merging.

File: eng/pipelines/create-doc-index.yml (hunk 1 above)
Comment (JimSuplizio) [reply eng/pipelines/create-doc-index.yml]: @weshaggard the java repo ran into this issue back in [December ](https://github.com/Azure/azure-sdk-for-java/pull/6925/files). On windows, it doesn't escape the backslashes correctly so we end up getting that messed up directory.

**Comment by scbedd** [comment]
@weshaggard or @JimSuplizio can I please get a re-signoff? I added changes to account for 404s from index link checker.

of the 4 errors from the [link checker run:](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=225016&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=e73d691f-7b26-5ce3-d518-700413f3ba2c)

1. Missing py-modindex.html. Since the index doesn't actually scan python modules, this is correct. Removed the link.
2. 404 not found on `roboto-slab` font. Added a css override to change the `@font-face` directive.
3. Missing document. Updated to point at new location for same content.
4. Too many redirects. Updated to point at final destination.

**Comment by weshaggard** [comment]
@scbedd did you regenerate the doc index? It looks like some of the links are still broken in our latest link validation run https://dev.azure.com/azure-sdk/internal/_build/results?buildId=227383&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=e73d691f-7b26-5ce3-d518-700413f3ba2c&l=85.

**Comment by scbedd** [comment]
@weshaggard You're correct. I had disabled autorelease to do one last check of the generated index (off of master) before enabling the final push to `gh-pages`. Everything looks fine to me (famous last words) and I let that go to github.io. Tonight's link checker run should reflect the new reality.

**Comment by weshaggard** [comment]
Cool last night's job is better https://dev.azure.com/azure-sdk/internal/_build/results?buildId=228827&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=e73d691f-7b26-5ce3-d518-700413f3ba2c&l=39. Only one missing font link left. Did you plan to fix that now or fix it when you convert the doc indexing?

**Pull request merged**